### DISCUSSION TITLE
Unable to load quantized base model with q5_0 in Linux centos machine

### DISCUSSION THREAD

**Discussion by ashish1garg** [ggml-org/whisper.cpp#2273] [Q&A]
I quantized the base model with q5_0. It got quantized to 55.3 MB. I am getting this error loading the model
"whisper_model_load: ERROR not all tensors loaded from model file - expected 245, got 3
whisper_init_no_state: failed to load model"
How can i make it work?

**Comment by candrews** [comment]
Has anyone found a solution to this problem that they can share?

**Comment by KitaitiMakoto** [comment]
How about to download quantized model and use it?

    % ./models/download-ggml-model.sh base-q5_1
    % ./build/bin/whisper-cli -m ./models/ggml-base-q5_1.bin -f ./samples/jfk.wav
